### PR TITLE
Update material-ui import paths

### LINKED ISSUE #2240: material-ui definitions are incorrect
Hi,

Material-ui is updated to version 1.0.0 and all of [its import paths have changed](https://github.com/mui-org/material-ui/pull/11330). The [current definitions](https://github.com/flowtype/flow-typed/blob/master/definitions/npm/%40material-ui/core_v1.x.x/flow_v0.58.x-/core_v1.x.x.js) are incorrect.

Thanks

### PR DESCRIPTION
**Pull request by KevinGrandon**
I actually haven't had much luck copying these into an app, but tests seem to be passing locally. Let's see what travis does.

Fixes #2240

### CODE REVIEW

File: definitions/npm/@material-ui/core_v1.x.x/test_core_v1.x.x.js [hunk 1]
```diff
@@ -3,10 +3,10 @@
 import React from "react";
 
 // Direct module import.
-import AppBar from "material-ui/AppBar/AppBar";
+import AppBar from "@material-ui/core/AppBar/AppBar";
```
Comment (TrySound) [inline definitions/npm/@material-ui/core_v1.x.x/test_core_v1.x.x.js]:
```js
import AppBar from "@material-ui/core/AppBar"
```

File: definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js [hunk 2]
```diff
@@ -2,51 +2,51 @@
...
-import Badge from "material-ui/Badge";
+import AppBar from "@material-ui/core/AppBar";
+import Avatar from "@material-ui/core/Avatar";
+import Badge from "@material-ui/core/Badge";
 import BottomNavigation, {
   BottomNavigationAction
-} from "material-ui/BottomNavigation";
-import Button from "material-ui/Button";
-import ButtonBase from "material-ui/ButtonBase";
+} from "@material-ui/core/BottomNavigation";
```
Comment (TrySound) [inline definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js]:
```js
import BottomNavigation from "@material-ui/core/BottomNavigation"
import BottomNavigationAction from "@material-ui/core/BottomNavigationAction"
```

File: definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js [hunk 3]
```diff
@@ -2,51 +2,51 @@
...
   CardActions,
   CardContent,
   CardHeader,
   CardMedia
-} from "material-ui/Card";
-import Checkbox from "material-ui/Checkbox";
-import Chip from "material-ui/Chip";
-import ClickAwayListener from "material-ui/utils/ClickAwayListener";
-import CssBaseline from "material-ui/CssBaseline";
+} from "@material-ui/core/Card";
```
Comment (TrySound) [inline definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js]:
```js
import Card from "@material-ui/core/Card"
import CardActions from "@material-ui/core/CardActions"
import CardContent from "@material-ui/core/CardContent"
import CardHeader from "@material-ui/core/CardHeader"
import CardMedia from "@material-ui/core/CardMedia"
```

File: definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js [hunk 4]
```diff
@@ -2,51 +2,51 @@
...
 import Dialog, {
   DialogActions,
   DialogContent,
   DialogContentText,
   DialogTitle,
   withMobileDialog
-} from "material-ui/Dialog";
-import Divider from "material-ui/Divider";
-import Drawer from "material-ui/Drawer";
+} from "@material-ui/core/Dialog";
```
Comment (TrySound) [inline definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js]:
```js
import Dialog from "@material-ui/core/Dialog"
import DialogActions from "@material-ui/core/DialogActions"
import DialogContent from "@material-ui/core/DialogContent"
import DialogContentText from "@material-ui/core/DialogContentText"
import DialogTitle from "@material-ui/core/DialogTitle"
import withMobileDialog from "@material-ui/core/withMobileDialog"
```

File: definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js [hunk 5]
```diff
@@ -2,51 +2,51 @@
...
+import Divider from "@material-ui/core/Divider";
+import Drawer from "@material-ui/core/Drawer";
 import ExpansionPanel, {
```
Comment (TrySound) [inline definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js]:
```js
import ExpansionPanel from "@material-ui/core/ExpansionPanel"
import ExpansionPanelActions from "@material-ui/core/ExpansionPanelActions"
import ExpansionPanelDetails from "@material-ui/core/ExpansionPanelDetails"
import ExpansionPanelSummary from "@material-ui/core/ExpansionPanelSummary"
```

File: definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js [hunk 6]
```diff
@@ -2,51 +2,51 @@
...
   FormHelperText,
   FormControlLabel
-} from "material-ui/Form";
-import Hidden from "material-ui/Hidden";
-import Icon from "material-ui/Icon";
-import IconButton from "material-ui/IconButton";
-import Input, { InputLabel, InputAdornment } from "material-ui/Input";
-import Grid from "material-ui/Grid";
-import GridList, { GridListTile, GridListTileBar } from "material-ui/GridList";
+} from "@material-ui/core/Form";
```
Comment (TrySound) [inline definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js]:
```js
import FormControl from "@material-ui/core/FormControl"
import FormGroup from "@material-ui/core/FormGroup"
import FormLabel from "@material-ui/core/FormLabel"
import FormHelperText from "@material-ui/core/FormHelperText"
import FormControlLabel from "@material-ui/core/FormControlLabel"
```

File: definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js [hunk 7]
```diff
@@ -2,51 +2,51 @@
...
   CardMedia
-} from "material-ui/Card";
-import Checkbox from "material-ui/Checkbox";
-import Chip from "material-ui/Chip";
-import ClickAwayListener from "material-ui/utils/ClickAwayListener";
-import CssBaseline from "material-ui/CssBaseline";
+} from "@material-ui/core/Card";
+import Checkbox from "@material-ui/core/Checkbox";
+import Chip from "@material-ui/core/Chip";
+import ClickAwayListener from "@material-ui/core/utils/ClickAwayListener";
```
Comment (TrySound) [inline definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js]:
```js
import ClickAwayListener from "@material-ui/core/ClickAwayListener"
```

File: definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js [hunk 8]
```diff
@@ -2,51 +2,51 @@
...
-import Input, { InputLabel, InputAdornment } from "material-ui/Input";
-import Grid from "material-ui/Grid";
-import GridList, { GridListTile, GridListTileBar } from "material-ui/GridList";
+} from "@material-ui/core/Form";
+import Hidden from "@material-ui/core/Hidden";
+import Icon from "@material-ui/core/Icon";
+import IconButton from "@material-ui/core/IconButton";
+import Input, { InputLabel, InputAdornment } from "@material-ui/core/Input";
+import Grid from "@material-ui/core/Grid";
+import GridList, { GridListTile, GridListTileBar } from "@material-ui/core/GridList";
```
Comment (TrySound) [inline definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js]:
```js
import GridList from "@material-ui/core/GridList"
import GridListTile from "@material-ui/core/GridListTile"
import GridListTileBar from "@material-ui/core/GridListTileBar"
```

File: definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js [hunk 9]
```diff
@@ -55,35 +55,35 @@ import List, {
...
-import MobileStepper from "material-ui/MobileStepper";
-import Modal, { Backdrop, ModalManager } from "material-ui/Modal";
-import Paper from "material-ui/Paper";
-import Popover from "material-ui/Popover";
-import Portal from "material-ui/Portal";
-import { CircularProgress, LinearProgress } from "material-ui/Progress";
-import Radio, { RadioGroup } from "material-ui/Radio";
-import Select from "material-ui/Select";
-import Snackbar, { SnackbarContent } from "material-ui/Snackbar";
+} from "@material-ui/core/List";
```
Comment (TrySound) [inline definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js]:
```js
import List from "@material-ui/core/List"
import ListItem from "@material-ui/core/ListItem"
import ListItemAvatar from "@material-ui/core/ListItemAvatar"
import ListItemIcon from "@material-ui/core/ListItemIcon"
import ListItemSecondaryAction from "@material-ui/core/ListItemSecondaryAction"
import ListItemText from "@material-ui/core/ListItemText"
import ListSubheader from "@material-ui/core/ListSubheader"
```

File: definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js [hunk 10]
```diff
@@ -55,35 +55,35 @@ import List, {
...
+import Menu, { MenuItem, MenuList } from "@material-ui/core/Menu";
```
Comment (TrySound) [inline definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js]:
```js
import Menu from "@material-ui/core/Menu"
import MenuItem from "@material-ui/core/MenuItem"
import MenuList from "@material-ui/core/MenuList"
```

File: definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js [hunk 11]
```diff
@@ -55,35 +55,35 @@ import List, {
...
+import MobileStepper from "@material-ui/core/MobileStepper";
+import Modal, { Backdrop, ModalManager } from "@material-ui/core/Modal";
+import Paper from "@material-ui/core/Paper";
+import Popover from "@material-ui/core/Popover";
+import Portal from "@material-ui/core/Portal";
+import { CircularProgress, LinearProgress } from "@material-ui/core/Progress";
```
Comment (TrySound) [inline definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js]:
```js
import CircularProgress from "@material-ui/core/CircularProgress"
import LinearProgress from "@material-ui/core/LinearProgress"
```

File: definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js [hunk 12]
```diff
@@ -55,35 +55,35 @@ import List, {
...
+import Radio, { RadioGroup } from "@material-ui/core/Radio";
```
Comment (TrySound) [inline definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js]:
```js
import Radio from "@material-ui/core/Radio"
import RadioGroup from "@material-ui/core/RadioGroup"
```

File: definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js [hunk 13]
```diff
@@ -55,35 +55,35 @@ import List, {
...
+import Select from "@material-ui/core/Select";
+import Snackbar, { SnackbarContent } from "@material-ui/core/Snackbar";
```
Comment (TrySound) [inline definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js]:
```js
import Snackbar from "@material-ui/core/Snackbar"
import SnackbarContent from "@material-ui/core/SnackbarContent"
```

File: definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js [hunk 14]
```diff
@@ -55,35 +55,35 @@ import List, {
...
 import Stepper, {
   Step,
   StepButton,
   StepIcon,
   StepContent,
   StepLabel
-} from "material-ui/Stepper";
+} from "@material-ui/core/Stepper";
```
Comment (TrySound) [inline definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js]:
```js
import Stepper from "@material-ui/core/Stepper"
import StepLabel from "@material-ui/core/StepLabel"
import StepButton from "@material-ui/core/StepButton"
import StepIcon from "@material-ui/core/StepIcon"
import StepContent from "@material-ui/core/StepContent"
import StepLabel from "@material-ui/core/StepLabel"
```

File: definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js [hunk 15]
```diff
@@ -92,12 +92,12 @@ import Table, {
...
   TableRow,
   TableSortLabel
-} from "material-ui/Table";
-import Tabs, { Tab } from "material-ui/Tabs";
-import Typography from "material-ui/Typography";
-import TextField from "material-ui/TextField";
-import Toolbar from "material-ui/Toolbar";
-import Tooltip from "material-ui/Tooltip";
-import { Slide, Grow, Fade, Collapse, Zoom } from "material-ui/transitions";
+} from "@material-ui/core/Table";
```
Comment (TrySound) [inline definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js]:
```js
import Table from "@material-ui/core/Table"
import TableBody from "@material-ui/core/TableBody"
import TableCell from "@material-ui/core/TableCell"
import TableFooter from "@material-ui/core/TableFooter"
import TableHead from "@material-ui/core/TableHead"
import TablePagination from "@material-ui/core/TablePagination"
import TableRow from "@material-ui/core/TableRow"
import TableSortLabel from "@material-ui/core/TableSortLabel"
```

File: definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js [hunk 16]
```diff
@@ -92,12 +92,12 @@ import Table, {
...
+import Tabs, { Tab } from "@material-ui/core/Tabs";
```
Comment (TrySound) [inline definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js]:
```js
import Tabs from "@material-ui/core/Tabs"
import Tab from "@material-ui/core/Tab"
```

File: definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js [hunk 17]
```diff
@@ -92,12 +92,12 @@ import Table, {
...
+import Typography from "@material-ui/core/Typography";
+import TextField from "@material-ui/core/TextField";
+import Toolbar from "@material-ui/core/Toolbar";
+import Tooltip from "@material-ui/core/Tooltip";
+import { Slide, Grow, Fade, Collapse, Zoom } from "@material-ui/core/transitions";
```
Comment (TrySound) [inline definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js]:
```js
import Slide from "@material-ui/core/Slide"
import Grow from "@material-ui/core/Grow"
import Fade from "@material-ui/core/Fade"
import Collapse from "@material-ui/core/Collapse"
import Zoom from "@material-ui/core/Zoom"
```

File: definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js [hunk 18]
```diff
@@ -92,12 +92,12 @@ import Table, {
...
 
-import withWidth from "material-ui/utils/withWidth";
+import withWidth from "@material-ui/core/utils/withWidth";
```
Comment (TrySound) [inline definitions/npm/@material-ui/core_v1.x.x/test_module_imports_core_v1.x.x.js]:
```js
import withWidth from "@material-ui/core/withWidth"
```

**Comment by KevinGrandon** [comment]
Let's go with #2255 instead, more activity over there.

**Pull request closed without merging**